includes/User.php

```diff
@@ -4218,6 +4218,20 @@ public static function crypt( $password, $salt = false ) {
 	 * @return Boolean
 	 */
 	public static function comparePasswords( $hash, $password, $userId = false ) {
+		// Wikia change - begin
+		// @see PLATFORM-2502 comparing new passwords in PHP code.
+		// @todo mech remove after the new password hashing is implemented (PLATFORM-2526).
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)